### ISSUE TITLE
Getting "order param is not in correct format" in Traces page

### ISSUE THREAD

**Issue by abhinavramana** [SigNoz/signoz#1748]
## Bug description

The traces page opens nicely, however on going to page 2 or 3 it gives "order param is not in correct format" consistently
![Actual error](https://user-images.githubusercontent.com/4195703/203620466-be24a8d6-cd27-4104-906f-eb1f9d28ee92.png)

![image](https://user-images.githubusercontent.com/4195703/203618458-b32f9989-e4fa-4a6b-9a04-7d10877d8309.png)
**Headers:**
```
Unexpected value 1rem parsing width attribute. main.7d1a4cf8b7f88ab77b0e.js:2:2057558
Unexpected value 1rem parsing height attribute. main.7d1a4cf8b7f88ab77b0e.js:2:2057558
Unexpected value 1rem parsing width attribute. main.7d1a4cf8b7f88ab77b0e.js:2:2057558
Unexpected value 1rem parsing height attribute. main.7d1a4cf8b7f88ab77b0e.js:2:2057558
XHRPOSThttps://signoz.wombo.net/api/v1/getFilteredSpans
[HTTP/1.1 400 Bad Request 40ms]

	
POST
	https://signoz.wombo.net/api/v1/getFilteredSpans
Status
400
Bad Request
VersionHTTP/1.1
Transferred410 B (114 B size)
Referrer Policystrict-origin-when-cross-origin
Request PriorityHighest

```

**Request**:
```
XHRPOST https://signoz.wombo.net/api/v1/getFilteredSpans
[HTTP/1.1 400 Bad Request 40ms]

	
end	"1669226910803000000"
exclude	[]
limit	10
maxDuration	""
minDuration	""
offset	10
order	"descending"
orderParam	""
start	"1669225110803000000"
tags	[]
```

**Stacktrace:**
```
XHRPOSThttps://signoz.wombo.net/api/v1/getFilteredSpans
[HTTP/1.1 400 Bad Request 40ms]

55448/e.exports/<
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:890350
55448/e.exports
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:888517
13572/e.exports
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:894257
(Async: promise callback) 30321/l.prototype.request
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:892519
30321/</l.prototype[e]
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:893096
91849/e.exports/<
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:898248
41183/t/<
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:27157
d
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:19601
41183/l/S/<
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:20939
41183/l/L/</<
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:19964
d
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:26288
i
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:27651
41183/e</y<
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:27710
41183/e<
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:27590
41183/y
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:27752
48159/p/e/<
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:184795
d
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:177939
48159/u/S/<
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:179277
48159/u/L/</<
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:178302
f
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:183931
i
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:185973
48159/p/r</<
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:186032
48159/p/r<
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:185912
48159/p/<
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:186076
53894/n/</</<
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:2343553
14890/d/<
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:2346269
88468/S/<
https://signoz.wombo.net/Trace Filter Page.d1727b032a6343d2e66b.js:2:162070
pu
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:2158302
60053/t.unstable_runWithPriority
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:2507108
wi
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:2098952
lu
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:2157765
Xc
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:2149293
64448/Oi/<
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:2099175
60053/t.unstable_runWithPriority
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:2507108
wi
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:2098952
Oi
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:2099122
xi
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:2099055
We
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:2169893
sn
https://signoz.wombo.net/main.7d1a4cf8b7f88ab77b0e.js:2:2077712
```

**Response**:
```
XHRPOSThttps://signoz.wombo.net/api/v1/getFilteredSpans
[HTTP/1.1 400 Bad Request 40ms]

	
data	null
total	0
limit	0
offset	0
errors	[ {…} ]
0	Object { code: 400, msg: "order param is not in correct format" }
```
## Expected behavior
No errors should pop up while moving through various pages in Traces section

## How to reproduce

1. Goto Traces page
2. Select page 2 or 3

## Version information 
* **Signoz version**: v0.11.3
* **Browser version**: Mozilla Firefox 
* **Your OS and version**: Ubuntu 22.04
* **Your CPU Architecture**(ARM/Intel): Intel

**Comment by dchennaraidu** [comment]
Hope this screenshot might help 

<img width="2560" alt="image" src="https://user-images.githubusercontent.com/12108102/203718217-9517596b-2ef3-41ec-ba5c-2c0bb4ddc168.png">

<img width="1721" alt="image" src="https://user-images.githubusercontent.com/12108102/203718647-afb92432-f81f-41f9-84b0-06adad223ae2.png">


https://github.com/SigNoz/signoz/blob/78d23775200ba8b8b2b84d09d09d734e90208831/pkg/query-service/app/parser.go#L280-L286

**Comment by makeavish** [comment]
@dchennaraidu Thanks for more information.
Have created a PR for this issue. Will be released in v0.11.4 release.
